frontend/src/pages/Admin/Components/AddTeamMember/AddTeamMember.jsx

```diff
@@ -131,6 +145,8 @@ export function AddTeamMember() {
       console.log(errors);
     } else {
       //Call the Server
+      const teamMemberData = { picUrl: picUrl, teams: teams, ...formdata };
```

Comment:
The picture won't be given in the form of url, you will have to write a function to convert the image in binary form (please keep it in util/shared folder so that it can be used by other components as well)